frappe/public/js/legacy/form.js

```diff
@@ -919,12 +919,23 @@ _f.Frm.prototype.add_custom_button = function(label, fn, group) {
 	return btn;
 };
 
+//Remove all custom buttons
 _f.Frm.prototype.clear_custom_buttons = function() {
 	this.page.clear_inner_toolbar();
 	this.page.clear_user_actions();
 	this.custom_buttons = {};
 };
 
+//Remove specific custom button by button Label
+_f.Frm.prototype.remove_custom_buttons = function(buttons) {
+	if(buttons){
+		for (var i=0; i<buttons.length; i++) {
+			if($.isArray(buttons[i]))
+				this.page.remove_inner_button(buttons[i][0], buttons[i][1]);
```

Comment:
what are these 2 parameters?

Reply:
these are (label, group), updated code to b clear